scenario_player/setup/nodes/config.py

```diff
@@ -0,0 +1,139 @@
+"""Helper module for configuring, starting and stopping Raiden Clients."""
+import pathlib
+import socket
+from typing import TYPE_CHECKING, Optional
+
+import structlog
+from eth_utils.address import to_checksum_address
+
+from scenario_player.setup.nodes.utils import create_keystore
+
+if TYPE_CHECKING:
+    from eth_utils.typing import ChecksumAddress
+    from scenario_player.setup.nodes.utils import RaidenBinary
+    from scenario_player.scenario import ScenarioYAML
+
+log = structlog.getLogger(__name__)
+
+
+class RaidenConfig:
```

Comment:
>I would suggest the the property names should as much as possible match the raiden cli option names. Otherwise this will quickly lead to confusion and bugs.

I can agree with that

>Also it may be useful to reuse parts of the existing implementation from scenario_player.node_support to not spend time re-implementing already working code.

As this is a refac, I wanted to first make sure I understand the code that's present by writing these docs. Since I'll be writing tests first, the existing code may not always fit the bill, but I'll make sure to start my implementation off of it where applicable.